### ISSUE TITLE
how to disable the remove functionallity in bootgrid

### ISSUE THREAD

**Issue by sushant357** [rstaib/jquery-bootgrid#162]
I am using pure html page i dont want to configure in formatting of bootgrid or somewhere else
if my code is like this
<td><div class="sample">Sample</div></td>
its shows me result like this
<td>Sample</td>
if i dont want to remove my div or any element how its possible...
please let me know...

**Comment by delboy1978uk** [comment]
Yes I completely agree. It's 'nice' that we have this formatter functionality, but stripping out HTML and forcing us to rewrite everything to use said formatter is just not on. Isn't there a way of disabling the formatter? And if not, could you add the feature?
